config/build.yml

```diff
@@ -5,14 +5,17 @@ kind: Sources
 sources:
 - image: kapp-controller
   path: .
-  #@ if/end not data.values.image_cache:
-  docker:
+  kubectlBuildkit:
```

Comment:
this was a really neat find @benmoss . I like that we can keep kbld but also use more functionality that we need. 💯